packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Add an external Identity Provider
```

Comment:
Would it make sense to work the IdP name into the page title for SEO and clarity of which IdP you're looking at since we no longer have the stack selector?  Is that possible with our setup?

Reply:
just tried that - it doesn't work with our set up. I can add the IdP name to the first sentence - would that help? Or no.